openshift/openshift/src/main/java/org/arquillian/cube/openshift/impl/client/PortForwarder.java

```diff
@@ -46,13 +29,29 @@
 import org.xnio.Xnio;
 import org.xnio.XnioWorker;
 import org.xnio.channels.AcceptingChannel;
-import org.xnio.channels.StreamSinkChannel;
 import org.xnio.ssl.XnioSsl;
 
+import javax.net.ssl.TrustManager;
+import javax.net.ssl.X509TrustManager;
+import java.io.Closeable;
+import java.io.IOException;
+import java.net.Inet4Address;
+import java.net.InetAddress;
+import java.net.InetSocketAddress;
+import java.net.URI;
+import java.net.UnknownHostException;
+import java.nio.ByteBuffer;
+import java.security.cert.X509Certificate;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.atomic.AtomicInteger;
+
 public final class PortForwarder implements Closeable {
 
     private static final String PORT_FWD = "%sapi/v1/namespaces/%s/pods/%s/portforward";
 
+    private static InetAddress portForwardBindAddress;
```

Comment:
This shouldn't not be static. Different objects should be able to listen on different addresses.